cache/build.gradle

```diff
@@ -60,6 +59,7 @@ dependencies {
     testImplementation cacheTestDependencies.junit
     testImplementation cacheTestDependencies.kotlinJUnit
     testImplementation cacheTestDependencies.mockito
+    testImplementation cacheTestDependencies.kotlinReflect
```

Comment:
I do not think that this library is necessary. Just curious - why did you add it in every module?

Reply:
This is so long ago, I'm not sure any more... Maybe it has to do with a gradle warning saying that different versions of a lib were found on the classpath. Maybe it's something else. Feel free to remove it. If it works well then I guess it's not needed !